bot/exts/evergreen/githubinfo.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands.cooldowns import BucketType
+
+log = logging.getLogger(__name__)
+
+
+class GithubInfo(commands.Cog):
+    """Fetches GitHub."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    async def fetch_data(self, url: str) -> dict:
+        """Retrieve data as a dictionary."""
+        async with aiohttp.ClientSession() as cs:
+            async with cs.get(url) as r:
+                return await r.json()
+
+    @commands.command(name='github', aliases=['gh'])
+    @commands.cooldown(1, 5, BucketType.user)
+    async def get_github_info(self, ctx: commands.Context, username: str = 'python-discord') -> None:
+        """
+        Fetches a user's GitHub information.
+
+        User defaults to python-discord.
+
+        Usage: .gh [username]
+        """
+        # fetch user data
+        user_data = await self.fetch_data(f"https://api.github.com/users/{username}")
+
+        # user_data will not have a message key if the user exists
+        if user_data.get('message') is not None:
+            return await ctx.send(embed=discord.Embed(title=f"The profile `{username}` was not found", colour=0xff0022))
```

Comment:
```suggestion
            return await ctx.send(embed=discord.Embed(title=f"The profile for `{username}` was not found.",
                                                      colour=0xff0022))
```
Added period and slightly altered the sentence.